src/components/header/Header.tsx

```diff
@@ -1,56 +1,55 @@
 'use client';
 
+import { Configuration } from '@/schemas/configuration-schema';
 import Image from 'next/image';
 import LocaleSwitcher from './LocaleSwitcher';
 import LogoKarlsruhe from '../../public/Logo_Digital_Karlsruhe-rechts_rgb.svg';
 import LogoKarlsruheSmall from '../../public/Logo_Digital_Karlsruhe-rechts_rgb_small.svg';
+import Search from '../search/Search';
 import { StaticImport } from 'next/dist/shared/lib/get-img-props';
 import { usePathname } from 'next/navigation';
 import { useShowNavigation } from '../navigation/NavigationProvider';
 import { useTranslations } from 'next-intl';
 
-export default function Header() {
+export default function Header({ configuration }: { configuration: Configuration }) {
   const { show, setShow } = useShowNavigation();
   const t = useTranslations('Header');
   const showSearchbar = !usePathname().endsWith('/home');
   return (
-    <div className="bg-white pb-3">
-      <div className="container-md">
-        <div className="d-flex flex-row d-md-none fs-5 m-2">
+    <div className="bg-white py-3">
+      <div className="container-lg">
+        <div className="d-flex justify-content-center m-2">
           <button
-            className="d-block btn btn-secondary"
-            onClick={(e) => {
+            className="d-block d-lg-none btn btn-secondary"
+            onClick={() => {
               setShow(!show);
             }}
           >
             <i className="bi bi-list" />
           </button>
-          <div className="d-block flex-fill align-content-center p-2 ps-4">{t('title')}</div>
+          <div className="d-block d-lg-none flex-fill align-content-center p-2 ps-4 fs-5">{t('title')}</div>
```

Comment:
I don't like the title and the searchbar existing twice, especially since the differences between the dual implementations are not that big. I think in these cases, we should try writing our own css classes to apply the changes at a given breakpoint.

Reply:
For the title I agree. I have fixed it by making the `fs` class responsive. For the searchbar, I don't think it is really possible  because it has to be in a different place within the markup (in its own container)